apiary.apib

```diff
@@ -344,9 +344,8 @@ We list all backwards-incompatible changes here. As described above, new additio
   `invoices.info`  
   and `creditNotes.info`.
 
-- Added the bookyear name to the `invoice_number` on `invoices.list` and `invoices.info`.
-  Earlier it was only returning the number, e.g. `"27"`.
-  Now it adds the name of the bookyear as well, e.g. `"2018 / 27`".
+- Invoice number actually contain the bookyear, so from now on, you will see eg. `2018 / 27` instead of `27`.
```

Comment:
Invoice numberS (plural here) or use "contains"

Reply:
solved it differently : `An invoice number actually contains the bookyear, ...`